client/src/templates/Challenges/classic/Editor.tsx

```diff
@@ -1,5 +1,4 @@
-import React, { Component, Suspense } from 'react';
```

Comment:
This is comment on the whole file, not this line.

I appreciate that this is probably heaps better than my implementation, but there are quite a few changes in this file so it's hard to tell what's just part of the migration to TS and what're bug fixes. Is there any way to split those into two commits?

I know that's not trivial, so I understand if the answer is "no, not really".

Reply:
Sure, I could do that.